src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -61,6 +61,32 @@ public static bool IsZipDeployment(this IEnvironment environment)
                 !string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteRunFromPackage));
         }
 
+        public static bool IsAppServiceWindowsEnvironment(this IEnvironment environment)
+        {
```

Comment:
This is only used in IsPersistentStorageAvailable. Remove this method and check in the method directly

Reply:
Related PR https://github.com/Azure/azure-functions-core-tools/pull/978